src/index.js

```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```

Comment:
- [ ] I download the repo, then I go to `main` branch, delete the `package-lock.json` file and then I use `npm i` to install the packages, after that I use `npm start` to check the deploy and this error appears, please work on that. 🐶 
![image](https://user-images.githubusercontent.com/20563252/128717823-6d271069-caaa-46a9-a468-5ed619b8040f.png)

Reply:
Thank you for the comment, it'll be fix soon.